### PR TITLE
clean up requested snomed codes, group all requested codes in one CS

### PR DESCRIPTION
**Pull request by saulakravitz**
- included all SCT codes that have been published in US edition
- include all SCT codes that have been published in Int'l edition in the REquestedSNOMEDCode CS
- Updated the request ID for Treatment plan.   The original request was rejected because I lost track of it...sigh... I've submitted a new request that addresses (hopefully) all of the objections.
- There are still 3 codes that need to be requested.

### CODE REVIEW

File: input/fsh/DEF_CodeSystems.fsh [hunk 1]
```diff
@@ -1,87 +1,23 @@
...
-* ^caseSensitive = true
-* ^experimental = false
-* #ring	"Ring, device (physical object)" 	"new concept under 228766009 |Form of brachytherapy source (physical object)"
-* #tandem "Tandem, device (physical object)" "(small metal tube)  new code child of 228775006 |Tubes source (physical object)."
-* #ovoid 	"Ovoid, device (physical object)" 	"(round hollow metal holders that are placed adjacent to cervix) new concept under 228766009 |Form of brachytherapy source (physical object)|"
-* #cylinder  "Vaginal Cylinder, device (physical object)" "A vaginal cylinder is made of plastic and looks like a large tampon with a hollow center. It is placed into the vagina and may be kept in place with gauze, balloons, or a special undergarment.  )  define a new concept under 228766009 |Form of brachytherapy source (physical object)| ."
+//* Requested 10/2022
+* #USCRS-34602  "Radiotherapy Treatment Plan (therapy/regime)" "Radiotherapy Treatment Plan (therapy/regime)"
+//* These haven't been requested
+* #USCRS-34603	"Changes to Treatment Intent (finding)" "new concept under 373856008 |Reason for change in radiotherapy course (finding)" //#treatmentintentchange
```
Comment (mvonsieb) [inline input/fsh/DEF_CodeSystems.fsh]: See latest code display strings in Sharon's mail from the workgroup call: "Change in treatment intent", "Change in target", "Change in organ at risk". We should already use lower case as usual in SNOMED CT.

File: input/fsh/DEF_CodeSystems.fsh [hunk 2]
```diff
@@ -1,87 +1,23 @@
...
+//* March 2023 release to US Edition
+* #1255213001 "Brachytherapy eye plaque (physical object)" "Brachytherapy eye plaque (physical object)"
+* #1255212006 "Brachytherapy surface applicator (physical object)" "Brachytherapy surface applicator (physical object)"
+* #1254780004 "Brachytherapy balloon (physical object)" "Brachytherapy balloon (physical object)"
+* #1254779002 "Brachytherapy catheter (physical object)"  "Brachytherapy catheter (physical object)"
+* #1254778005 "Brachytherapy ring (physical object)" "Brachytherapy ring (physical object)"
+* #1254777000 "Brachytherapy ovoid (physical object)" "Brachytherapy ovoid (physical object)"
+* #1254776009 "Brachytherapy tandem (physical object)" "Brachytherapy tandem (physical object)"
+* #1255211004 "Brachytherapy cylinder (physical object)" "Brachytherapy cylinder (physical object)"
+* #1254768007 "Radioactive seed embedded in mesh (physical object)" "Radioactive seed embedded in mesh (physical object)"
```
Comment (mvonsieb) [inline input/fsh/DEF_CodeSystems.fsh:22]: Plus a code for Loose seed?

File: input/fsh/DEF_CodeSystems.fsh [hunk 3]
```diff
@@ -1,87 +1,23 @@
...
-Description: "Brachytherapy Applicator Types  -- need to request codes"
-* ^status = #draft
-* ^caseSensitive = true
-* ^experimental = false
-* #ring	"Ring, device (physical object)" 	"new concept under 228766009 |Form of brachytherapy source (physical object)"
-* #tandem "Tandem, device (physical object)" "(small metal tube)  new code child of 228775006 |Tubes source (physical object)."
-* #ovoid 	"Ovoid, device (physical object)" 	"(round hollow metal holders that are placed adjacent to cervix) new concept under 228766009 |Form of brachytherapy source (physical object)|"
-* #cylinder  "Vaginal Cylinder, device (physical object)" "A vaginal cylinder is made of plastic and looks like a large tampon with a hollow center. It is placed into the vagina and may be kept in place with gauze, balloons, or a special undergarment.  )  define a new concept under 228766009 |Form of brachytherapy source (physical object)| ."
+//* Requested 10/2022
+* #USCRS-34602  "Radiotherapy Treatment Plan (therapy/regime)" "Radiotherapy Treatment Plan (therapy/regime)"
```
Comment (mvonsieb) [inline input/fsh/DEF_CodeSystems.fsh:8]: Already use lower case as in SNOMED 1255724003 "Radiotherapy treatment plan (regime/therapy)"

File: input/fsh/SD_RadiotherapyCoursePrescription.fsh [hunk 4]
```diff
@@ -15,5 +15,5 @@ Description: "A Radiotherapy Course Prescription is a high-level request for a c
   * extension[fractionDose] ^definition = "Fraction dose is not defined in the Course Prescription because dose may not be equal in all Fractions in which this volume receives dose."
 * occurrencePeriod only Period
 * occurrencePeriod MS
-* code = $mCODESCT_TBD#USCRS-33529 "Radiotherapy Course of Treatment (regime/therapy)"
+* code = SCT#1217123003  "Radiotherapy Course of Treatment (regime/therapy)"
```
Comment (mvonsieb) [inline input/fsh/SD_RadiotherapyCoursePrescription.fsh:18]: Lower case: "Radiotherapy course of treatment (regime/therapy)"

File: input/fsh/SD_RadiotherapyPhasePrescription.fsh [hunk 5]
```diff
@@ -6,7 +6,7 @@ Title: "Radiotherapy Phase Prescription"
 Description: "A Radiotherapy Phase Prescription is a request for one series of fractionated treatments using Radiotherapy. It can define treatment parameters such as modality or technique,
 and prescribe dose to multiple targets."
 * insert RadiotherapyPhaseAndPlanPrescriptionCommon
-* code = SnomedRequestedCS#USCRS-33527 "Radiotherapy Treatment Phase (therapy/regime)"
+* code = SCT#1222565005 "Radiotherapy Treatment Phase (therapy/regime)"
```
Comment (mvonsieb) [inline input/fsh/SD_RadiotherapyPhasePrescription.fsh]: lower case and regime is first in the bracket "Radiotherapy treatment phase (regime/therapy)".

File: input/fsh/VS_ReasonForRevisitionOrAdaptation.fsh [hunk 6]
```diff
@@ -0,0 +1,11 @@
+ValueSet: ReasonForRevisionOrAdaptationVS
+Id: codex-radiotherapy-reason-for-revision-or-adaptation-vs
+Title: "Reason for Revision or Adaptation Value Set"
+Description: "The reason a planned or prescribed radiotherapy treatment was revised, superceded, or adapted."
+* insert SCTCopyright
+* ^experimental = false
+* SCT#373858009 "Radiotherapy course changed - acute radiotherapy toxicity (finding)"
+* SCT#314846003 "Patient given choice of treatment (finding)"
+* SnomedRequestedCS#USCRS-34603	"Changes to Treatment Intent (finding)"
```
Comment (mvonsieb) [inline input/fsh/VS_ReasonForRevisitionOrAdaptation.fsh]: New display strings. See coding system.

**Review by mvonsieb — APPROVED** [review]

File: input/fsh/DEF_CodeSystems.fsh (hunk 1 above)
Comment (saulakravitz) [reply input/fsh/DEF_CodeSystems.fsh]: done

File: input/fsh/DEF_CodeSystems.fsh (hunk 3 above)
Comment (saulakravitz) [reply input/fsh/DEF_CodeSystems.fsh:8]: Loose seed is an existing code:
SCT#1254766006 "Loose radioactive seed (physical object)"

**Pull request merged**